### ISSUE TITLE
Layout is same for Post and Page

### ISSUE THREAD

**Issue by varunchandak** [klugjo/hexo-theme-clean-blog#33]
Looking into the layout/ folder of **clean-blog** theme, I found that creating a new page and a post resulted into same layout. This is because **post.ejs** and **page.ejs** has same content.

Due to this, I wasn't unable to make a new Page of **About Me** for my blog.

**Comment by klugjo** [comment]
Yes posts and pages have the same layout by default, but that should not prevent you from creating pages.

Can you give more details on the issue you are facing and the steps to reproduce ?

**Comment by varunchandak** [comment]
Currently, my blog "cloudbeings.in" has 3 pages.
When i ran the command: **hexo new page About**, it created the page, but it also had the alignment to left, and date underneath the heading.

This is similar to the a sample blog post, if you check.

But if you look at home page or Blog page, the header text is center aligned, and subtitle instead of date.

I fixed the about page by creating About.ejs file under **themes/clean-blog/layout/About.ejs** and in the **source/About/index.md** I changed 
the layout format to About **layout: About**
